cmd/crio/main.go

```diff
@@ -269,6 +269,26 @@ func main() {
 			return err
 		}
 
+		// Print the current CLI flags.
+		for _, flagName := range c.FlagNames() {
+			flagValue := c.Value(flagName)
+			// Turn a multi-value flag into a single command-separated list
```

Comment:
Bummer! I made this typo, apologies!

```suggestion
			// Turn a multi-value flag into a single comma-separated list
```